packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -332,38 +353,27 @@ const AccountSwitcher = props => {
         return !!(props.is_virtual && props.can_upgrade_to);
     };
 
-    const getTotalBalance = (accounts, is_demo = true) => {
+    const getTotalBalanceCfd = (accounts, is_demo, exchange_rate) => {
         return accounts
             .filter(account => (is_demo ? isDemo(account) : !isDemo(account)))
             .reduce(
                 (total, account) => {
-                    const real_account_loginid = props.account_list?.find(acc => !acc.is_virtual)?.loginid;
-                    if (!is_demo && props.accounts[real_account_loginid]?.currency !== account.currency) {
-                        total.balance += account.balance * exchanged_rate;
-                    } else {
-                        total.balance += account.balance;
-                    }
+                    total.balance += account.balance * exchange_rate;
                     return total;
                 },
                 { balance: 0 }
             );
     };
 
     const getTotalDemoAssets = () => {
-        const vrtc_loginid = props.account_list.find(account => account.is_virtual).loginid;
         const vrtc_balance = props.accounts[vrtc_loginid] ? props.accounts[vrtc_loginid].balance : 0;
-        const vrtc_currency = props.accounts[vrtc_loginid] ? props.accounts[vrtc_loginid].currency : 'USD';
-        const mt5_demo_total = getTotalBalance(props.mt5_login_list);
-        const dxtrade_demo_total = getTotalBalance(props.dxtrade_accounts_list);
+        const mt5_demo_total = getTotalBalanceCfd(props.mt5_login_list, true, exchanged_rate_cfd_demo);
+        const dxtrade_demo_total = getTotalBalanceCfd(props.dxtrade_accounts_list, true, exchanged_rate_cfd_demo);
 
-        let total = vrtc_currency !== props.obj_total_balance.currency ? vrtc_balance * exchanged_rate : vrtc_balance;
-
-        if (Array.isArray(props.mt5_login_list)) {
-            total += mt5_demo_total.balance;
-        }
-        if (Array.isArray(props.dxtrade_accounts_list)) {
-            total += dxtrade_demo_total.balance;
-        }
+        const total =
+            (vrtc_currency !== account_total_balance_currency ? vrtc_balance * exchanged_rate_demo : vrtc_balance) +
```

Comment:
Things look good to me, just a big tip for readability. If you write long ternaries/statements like this, it will help a lot if you combine parts of them in variables to make it easier to read. Otherwise, other people who will read your code will take a long time to fully understand this line.

An example of what I mean: 
```
const account_is_vrtc_currency = vrtc_currency === account_total_balance_currency;
const vrtc_balance_exchanged = vrtc_balance * exchanged_rate_demo;

(!account_is_vrtc_currency ? vrtc_balance_exchanged : vrtc_balance) +
...
```

I do not know exactly how to name the variables in this case (this is the hardest part, haha). But, I hope you get the idea.

Reply:
Yeah, I got your idea. Thanks) will keep in mind it ;)